submarine-commons/commons-unixusersync/src/test/java/org/apache/submarine/commons/unixusersync/jwtToken.java

```diff
@@ -0,0 +1,64 @@
+package org.apache.submarine.commons.unixusersync;
+
+
+import javax.crypto.spec.SecretKeySpec;
+import javax.xml.bind.DatatypeConverter;
+import java.security.Key;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.JwtBuilder;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.impl.crypto.MacProvider;
+
+import java.util.Base64;
+import java.util.Date;
+
+public class jwtToken {
+
+  private static final Key secret = MacProvider.generateKey(SignatureAlgorithm.HS256);
+  private static final byte[] secretBytes = secret.getEncoded();
+  private static final String base64SecretBytes = Base64.getEncoder().encodeToString(secretBytes);
+
+  //Sample method to construct a JWT
+
+  public static String createJWT(String id, String issuer, String subject, long ttlMillis) {
+    //The JWT signature algorithm we will be using to sign the token
```

Comment:
The comment needs to be left blank. like, `// Sample method to construct a JWT`
The same goes for other places